rust/otel-arrow-rust/src/proto/opentelemetry.proto.experimental.arrow.v1.rs

```diff
@@ -306,18 +307,26 @@ pub mod arrow_traces_service_client {
             tonic::Response<tonic::codec::Streaming<super::BatchStatus>>,
             tonic::Status,
         > {
-            self.inner.ready().await.map_err(|e| {
-                tonic::Status::unknown(format!("Service was not ready: {}", e.into()))
-            })?;
+            self.inner
```

Comment:
Is this related to markdownlint or it is accidentally included in the PR?

Reply:
Thanks, that was a mistake.